sql/updates/0.10.1--0.11.0-dev.sql

```diff
@@ -0,0 +1,5 @@
+DROP FUNCTION IF EXISTS drop_chunks(INTERVAL, NAME, NAME, BOOLEAN);
+DROP FUNCTION IF EXISTS drop_chunks(ANYELEMENT, NAME, NAME, BOOLEAN);
+DROP FUNCTION IF EXISTS _timescaledb_internal.drop_chunks_impl(BIGINT, NAME, NAME, BOOLEAN, BOOLEAN);
+DROP FUNCTION IF EXISTS _timescaledb_internal.drop_chunks_type_check(REGTYPE, NAME, NAME);
+DROP FUNCTION IF EXISTS _timescaledb_internal.dimension_get_time(INTEGER);
```

Comment:
depending on which release this function ends up in, we should remember to put this in relevant update file.
Also if we plan a 0.12.0 release, this file should be renamed right @RobAtticus ?

Reply:
You may have to rebase and move it there manually, not entirely sure how this will be handled. good job surfacing it early though